.github/ci-test-connections.yaml

```diff
@@ -35,3 +35,19 @@ connections:
     schema:
     login:
     password:
+  - conn_id: gcp_conn
+    conn_type: google_cloud_platform
+    description: null
+    extra: null
+  - conn_id: google_cloud_default
+    conn_type: google_cloud_platform
+    description: null
+    extra: null
+  - conn_id: aws_conn
+    conn_type: aws
+    description: null
+    extra: null
+  - conn_id: aws_default
```

Comment:
The same question on `google_cloud_default` applies here

Reply:
Fixed.